README.md

```diff
@@ -1,24 +1,96 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Knead Recipies Logo](https://i.imgur.com/RTE6Gh1.png)
```

Comment:
Mike has a banner version that might suite better! [https://turingschool.slack.com/files/UQTT2KJMC/F01516HE4NQ/knead_recipes.png](url)